src/selic-entries.ts

```diff
@@ -0,0 +1,12 @@
+export async function selicEntries(initialDate: string, finalDate: string) {
+    const url = new URL('https://api.bcb.gov.br/dados/serie/bcdata.sgs.11/dados?formato=json');
+    const params = new URLSearchParams([
+        ['dataInicial', initialDate],
+        ['dataFinal', finalDate],
+    ]);
+    url.search = params.toString();
```

Comment:
While this syntax you've chosen is ok, it doesn't look as good as using a hash. See the [examples](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams#examples) below:

```javascript
// Retrieve params via url.search, passed into constructor
const url = new URL("https://example.com?foo=1&bar=2");
const params1 = new URLSearchParams(url.search);

// Get the URLSearchParams object directly from a URL object
const params1a = url.searchParams;

// Pass in a string literal
const params2 = new URLSearchParams("foo=1&bar=2");
const params2a = new URLSearchParams("?foo=1&bar=2");

// Pass in a sequence of pairs
const params3 = new URLSearchParams([
  ["foo", "1"],
  ["bar", "2"],
]);

// Pass in a record
const params4 = new URLSearchParams({ foo: "1", bar: "2" });
```

---

```suggestion
    url.search = new URLSearchParams({
        dataInicial: initialDate,
        dataFinal: finalDate,
    }).toString();
```